### DISCUSSION TITLE
OpenMPTCPRouter with multiple public IPs

### DISCUSSION THREAD

**Discussion by bernhard2501** [Ysurac/openmptcprouter#2964] [Q&A]
hi there altogether, 

I am pretty new to OpenMPTCPRouter but I think it is a great piece of work to get multiple WAN connections bonded to one connection for better up- und downloads. Currently I am using it with two 4G modems/routers behind a VPS and it works great.

Therefore, I have a question: since 5G is much faster than dsl or cable in my region, I am thinking of putting my existing dsl /29 net with public ips and servers behind OpenMPTCPRouter and my Sophos. So to precise my question - is it possible to route all traffic from OpenMPTCPRouter to my exisiting Sophos and the more is it possible to get multiple public IP on my VPS, route them to my Sophos and then back to OpenMPTCPRouter - like I have it with my dsl right now?

maybe someone of you specialists can help me with that question? If this would work that would be a big advancement in my server structure!

thank you very much in advance, best regards, Bernhard

**Comment by Ysurac** [comment]
Hi,

If /29 is on you DSL no (or traffic will only use DSL line), if you have multiple public IPs on the VPS yes you can use them: https://github.com/Ysurac/openmptcprouter/wiki/VPS-Multi-IPs

**Comment by bernhard2501** [comment]
hi Yannick,
thanks for your quick reply - just for some clarification - so if I have an
public IP address like 37.252.189.73 on the vps and I would route it to
192.168.100.73 for instance with the help of openmptcpreouter - and the
192.168.100.73 is an interface of my sophos firewall (DMZ?) I could use my
firewall to reroute the traffic to my email server behind the firewall -
like 10.52.0.3 and backwards the same way over my sophos and routing to
your opemmptcprouter?
thanks again, bernhard

Am Do., 21. Sept. 2023 um 09:16 Uhr schrieb Ycarus (Yannick Chabanois) <
***@***.***>:
